packages/blaze/builtins.js

```diff
@@ -159,6 +252,8 @@ Blaze.Each = function (argFunc, contentFunc, elseFunc) {
             eachView.expandedValueDep.changed();
           } else if (eachView._domrange) {
             eachView._domrange.moveMember(fromIndex, toIndex);
+            updateIndexes(
```

Comment:
I will use updateIndicies. Since this function is updating a range of views, `updateAtIndex` sounds like it is updating a single value.